### PR TITLE
Avoid hanging for nodes in deleted/remote symbols

### PR DESCRIPTION
**Pull request by sicking**

### CODE REVIEW

File: src/helpers/isPartOfNode.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
 //this function allows you to check whether a node is part of an rootNode
-export default function isPartOfNode(part: SceneNode, rootNode: BaseNode): boolean {
-	const parent = part.parent
-	if (parent === rootNode) {
-		return true
-	} else if (parent.type === 'PAGE') {
-		return false
-	} else {
-		return isPartOfNode(parent as SceneNode, rootNode)
+export default function isPartOfNode(part: BaseNode?, rootNode: BaseNode): boolean {
```
Comment (Andreslav) [inline src/helpers/isPartOfNode.ts]: `?` after `BaseNode` provokes an error.

**Comment by sicking** [comment]
Ah, yeah, that should be `BaseNode | null`

**Comment by Andreslav** [comment]
@sicking Fix it?

**Comment by alexandrtovmach** [comment]
@sicking Could you apply requested changes?

**Comment by alexandrtovmach** [comment]
Closing as staled

**Pull request closed without merging**